src/pages/ListPage/ListHeader/ListMenu/SubMenus.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import FilterModal from '../components/FilterModal';
+import { SUB_MENUS } from './SubMenuData';
+
+export default function SubMenus(props) {
+  const [modal, setModal] = useState(false);
+
+  const handleClickModal = index => {
+    if (index === 3) {
+      setModal(true);
+    }
+  };
+
+  return (
+    <>
+      {SUB_MENUS.map((menu, idx) => {
+        return (
+          <FilterButton
+            key={idx}
+            onClick={() => {
+              handleClickModal(idx);
+            }}
+          >
+            {menu}
+          </FilterButton>
+        );
+      })}
+      {modal === true ? <FilterModal setModal={setModal} /> : null}
```

Comment:
```suggestion
      {modal && <FilterModal setModal={setModal} />}
```
우리는 이걸 단축평가로 해결하기로 했죠..